shared/constants.ts

```diff
@@ -7,8 +7,8 @@ export let MIMOTO_BASE_URL = MIMOTO_HOST;
 export let ESIGNET_BASE_URL = ESIGNET_HOST;
 export let DEBUG_MODE_ENABLED = DEBUG_MODE === 'true';
 
-export const changeCrendetialRegistry = host => (MIMOTO_BASE_URL = host);
-export const changeEsignetUrl = host => (ESIGNET_BASE_URL = host);
+export const changeCrendetialRegistry = (host: any) => (MIMOTO_BASE_URL = host);
+export const changeEsignetUrl = (host: any) => (ESIGNET_BASE_URL = host);
 
```

Comment:
As host will be always be string, so please have specific string type instead of any